website/docs/r/attestation.html.markdown

```diff
@@ -43,8 +43,20 @@ The following arguments are supported:
 
 -> **NOTE:** If the `policy_signing_certificate_data` argument contains more than one valid X.509 certificate only the first certificate will be used.
 
+* `policy` - (Optional) One or more `policy` blocks as defined below.
+
 * `tags` - (Optional) A mapping of tags which should be assigned to the Attestation Provider.
 
+---
+
+A `policy` supports the followings:
+
+* `type` - (Required) Specifies the type of the trusted environment to be used. Possible values are `OpenEnclave`, `SgxEnclave` and `Tpm`.
```

Comment:
this might be more clear as
```suggestion
* `environment_type` - (Required) Specifies the type of the trusted environment to be used. Possible values are `OpenEnclave`, `SgxEnclave` and `Tpm`.
```